model/db-common.yaml

```diff
@@ -0,0 +1,47 @@
+groups:
+  - id: attributes.db.common
+    type: attribute_group
+    brief: 'Describes Database attributes.'
+    attributes:
+      - ref: db.system
+        requirement_level: required
```

Comment:
#nit: Why have a separate group for `attributes.db.common` before without any idea of when db server semconv would be defined?

Reply:
This was not about server conventions. I originally made this common group because I thought I recalled that the existing [connection pool metrics](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-metrics.md#connection-pools) used the `db.system` attribute. Reviewing them again, it appears I misremembered.

Anyways... looks like I can remove this common group for now.